project_national/fact_sheets_category_6.yml

```diff
@@ -0,0 +1,752 @@
+schema_version: '0.3'
+os_version: 3.6.1
+buildstock_directory: ../ # Relative to this file or absolute
+project_directory: project_national # Relative to buildstock_directory
+output_directory: /projects/abctypol/largeee/test_runs/small_run_category_6_20230609/
+# weather_files_url: https://data.nrel.gov/system/files/156/BuildStock_TMY3_FIPS.zip
+weather_files_path: /shared-projects/buildstock/weather/BuildStock_TMY3_FIPS.zip
+
+sampler:
+  type: precomputed
+  args:
+    sample_file: /projects/abctypol/largeee/buildstocks/buildstock_100_20230609.csv #../buildstock_100_20230609.csv # relative to file
+
+baseline:
+  n_buildings_represented: 136569411 # American Community Survey 2019 5-year, B25001, does not include AK, HI, and territories
+  skip_sims: true
+
+eagle:
+  n_jobs: 1
+  minutes_per_sim: 4
+  account: abctypol
+  postprocessing:
+    time: 20
+    n_workers: 1
+  sampling:
+    time: 5
+
+postprocessing:
+  keep_individual_timeseries: false
+  aws:
+    region_name: us-west-2
+    s3:
+      bucket: largeee
+      prefix: test_runs
+    athena:
+      glue_service_role: service-role/AWSGlueServiceRole-default
+      database_name: largeee_test_runs
+      max_crawling_time: 1200
+
+workflow_generator:
+  type: residential_hpxml
+  args:
+    build_existing_model:
+      simulation_control_timestep: 60
+      simulation_control_run_period_begin_month: 1
+      simulation_control_run_period_begin_day_of_month: 1
+      simulation_control_run_period_end_month: 12
+      simulation_control_run_period_end_day_of_month: 31
+      simulation_control_run_period_calendar_year: 2007
+
+    emissions:
+      - scenario_name: LRMER_MidCase_15
+        type: CO2e
+        elec_folder: data/cambium/LRMER_MidCase_15
+
+    utility_bills:
+      - scenario_name: Bills
+
+    simulation_output_report:
+      timeseries_frequency: timestep #monthly
+      include_timeseries_total_consumptions: true
+      include_timeseries_fuel_consumptions: true
+      include_timeseries_end_use_consumptions: true
+      include_timeseries_emissions: true
+      include_timeseries_total_loads: false
+
+    reporting_measures:
+      - measure_dir_name: QOIReport
+
+    server_directory_cleanup:
+      retain_in_idf: false
+      retain_schedules_csv: false
+
+
+### -- Upgrades ---
+# Category 6:  Heat Pumps with Existing Backup
+# To do: costing interpolation from Less for 9.2, 9.5, and 11 HSPF
+upgrades:
+  - upgrade_name: EnergyStar ducted ASHP w/ existing backup  #also provides SEER 15 cooling measure profile
+    # Measure 1
+    options:
+      #ASHP
+      - &ducted_ASHP_EnergyStar_exist_backup_no_shared_ducts
+        option: HVAC Heating Efficiency|ASHP, SEER 16, 9.2 HSPF, Separate Backup # same as base option
+        apply_logic: &dualfuel_ASHP_no_ducted_furnace_apply_logic
+          - HVAC Has Ducts|Yes
+          - or:
+            - Heating Fuel|Natural Gas
+            - Heating Fuel|Propane
+            - Heating Fuel|Fuel Oil
```

Comment:
I was expecting these to be three separate upgrades - let's touch base separately on this.

Reply:
Please split out all HP upgrades by primary baseline heating fuel per the proposal spreadsheet.